_posts/2021-02-06-sr2021-second-league-sessions.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: news
+title: SR2021 Teams Show Progress in Second League Session
+---
+
+With another 36 matches of the SR2021 Competition, team HRS3 from Hills Road
+Sixth Form College head up the league leaderboard, though a number of other
+teams are hot on their heels. This weekend's matches saw great perfomances from
+our teams, including three new teams which hadn't submitted a robot for the
+first league sesion.
+
+Module Ⅱ provided a new challenge to the teams, both through opening up the
+arena half way through the match and by requiring that territories be linked
+back to a teams starting zone to be captured. The latter in particular created
+some tense moments during the matches, with robots able to dramatically affect
+each others' scores by severing territory chains.
+
+At the end of the league we announced to teams the details of our
+[third game module](https://studentrobotics.org/docs/resources/2021/rulebook.html).
+This changes the way that a robot's score is calculated from being the largest
+extent of their territory during the match to being purely based on the
+territory they control at the end of the match. Additionally the territories
+will now become locked (and unclaimed) after they have been successfully claimed
+four times.
+
+This module of the game requries that the teams robots further increase their
+awawreness of their environment and how they react to it if they want to claim
+territories and also hold them for the duration of the match
+
+We're now at the half-way point in the league matches that will seed the evntual
+knockout stage, so teams still have everything to play for. The next league
+session will be in 6 weeks, on the 20th March.
+
+If you would like to catch up on any of the matches so far, our livestreams are
+available on YouTube:
+
+* [League Session 2](https://www.youtube.com/watch?v=RwW5Oz30gbE)
+* [League Session 1](https://www.youtube.com/watch?v=cAvk-nfTUis)
+
+We look forward to seeing you over the coming months for the continuation of
+the SR2021 Competition.
+
+| Date              | Competition Stage                                                                     |
+|-------------------|---------------------------------------------------------------------------------------|
+| 20th March 2021   | [League Session 3]({{ site.baseurl }}/events/sr2021/league-3/)                        |
+| 24th April 2021   | [League Session 4]({{ site.baseurl }}/events/sr2021/league-4/)                        |
+| 1st May 2021      | [Knockouts and Final]({{ site.baseurl }}/events/sr2020/virtual-competition-knockouts/)|
```

Comment:
```suggestion
| 1st May 2021      | [Knockouts and Final]({{ site.baseurl }}/events/sr2021/knockouts/)|
```

Reply:
this was wrong in the previous post too it seems: 6ab2a70996b955be4b18b86eff018f9fef89d41a